src/webui/src/components/Modal/LogDrawer.tsx

```diff
@@ -139,7 +82,7 @@ class LogDrawer extends React.Component<LogDrawerProps, LogDrawerState> {
                     closable={false}
                     destroyOnClose={true}
                     onClose={closeDrawer}
-                    visible={isVisble}
+                    visible={true}
```

Comment:
why `visible` always is true？

Reply:
Because when it's not visible, `LogDrawer` will not render.